app/controllers/smart_proxies_controller.rb

```diff
@@ -6,7 +6,10 @@ class SmartProxiesController < ApplicationController
   before_action :find_status, :only => [:ping, :tftp_server]
 
   def index
-    @smart_proxies = resource_base_search_and_page.includes(:features)
+    # Large content_counts data may cause slow page load, especially with eager load.
```

Comment:
Does the api controller suffer from the same slowness?

Reply:
no. I think api controller doesn't do eager load.